string-hash/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for string-hash 1.1.1
+// Project: https://github.com/darkskyapp/string-hash
+// Definitions by: Ethan Rubio <https://github.com/ethanrubio/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace StringHash {
```

Comment:
This whole definition could just be:

```ts
function stringHash(str: string): number;
export = stringHash;
export as namespace stringHash;
```

(without any `declare module` wrapper)